src/oci-umount.c

```diff
@@ -580,6 +580,18 @@ static int parseBundle(yajl_val *node_ptr, char **rootfs, struct config_mount_in
 	}
 	*rootfs=strdup(YAJL_GET_STRING(v_root));
 
+	/* Prepend bundle path if the rootfs string is relative */
+	if (*rootfs[0] != '/') {
+		char *new_rootfs;
+
+		asprintf(&new_rootfs, "%s/%s", YAJL_GET_STRING(v_bundle_path), *rootfs);
+		if (!new_rootfs) {
+			pr_perror("failed to alloc rootfs");
+			return EXIT_FAILURE;
+		}
+		*rootfs = new_rootfs;
```

Comment:
It really should be the *rootfs = new_rootfs;

The compiler is complaining because the types don't match.  So we look at the the definition of the variable:

static int parseBundle(yajl_val *node_ptr, char **rootfs, struct config_mount_in

The char **rootfs means that we got the pointer to the pointer of rootfs.  This is done so someone can pass back a pointer to an array of characters.   In this case the one that just got allocated by the asprintf().  

You could also make it compile without a warning with:

rootfs = &new_rootfs;

But that is not going to do what you want.  That would just assign the local instance variable rootfs pointer a new value which is thown away after returning from the function.